page.codeberg.SOrg.DogGTK.json

```diff
@@ -0,0 +1,51 @@
+{
+    "app-id" : "page.codeberg.SOrg.DogGTK",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "doggtk",
+    "finish-args" : [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--device=dri"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args": [
+            "--share=network"
```

Comment:
this won't work it's not allowed.

the build must be offline

Reply:
Just removed it now :)